Makefile

```diff
@@ -79,45 +93,70 @@ _tag_version: current_version
-	CGO_ENABLED=0 GOOS=plan9     GOARCH=386      $(COMPILE_COMMAND) && mv ./bin/ec ./bin/ec-plan9-386
-	CGO_ENABLED=0 GOOS=plan9     GOARCH=amd64    $(COMPILE_COMMAND) && mv ./bin/ec ./bin/ec-plan9-amd64
-	CGO_ENABLED=0 GOOS=solaris   GOARCH=amd64    $(COMPILE_COMMAND) && mv ./bin/ec ./bin/ec-solaris-amd64
-	CGO_ENABLED=0 GOOS=windows   GOARCH=386      $(COMPILE_COMMAND) && mv ./bin/ec ./bin/ec-windows-386.exe
-	CGO_ENABLED=0 GOOS=windows   GOARCH=amd64    $(COMPILE_COMMAND) && mv ./bin/ec ./bin/ec-windows-amd64.exe
+	@# To generate list of supported targets, run:
+	@#  go tool dist list
+	$(call _build_target,aix/ppc64)
+	@# gcc.exe: error: unrecognized command-line option '-rdynamic'
+	@# $(call _build_target,android/386)
+	@# gcc.exe: error: unrecognized command-line option '-rdynamic'
+	@# $(call _build_target,android/amd64)
+	@# gcc.exe: error: unrecognized command-line option '-marm'; did you mean '-mabm'?
+	@# gcc.exe: error: unrecognized command-line option '-rdynamic'
+	@# $(call _build_target,android/arm)
+	$(call _build_target,android/arm64)
+	$(call _build_target,darwin/amd64)
+	$(call _build_target,darwin/arm64)
+	$(call _build_target,dragonfly/amd64)
+	$(call _build_target,freebsd/386)
+	$(call _build_target,freebsd/amd64)
+	$(call _build_target,freebsd/arm)
+	$(call _build_target,freebsd/arm64)
+	$(call _build_target,illumos/amd64)
+	@# gcc.exe: error: unrecognized command-line option '-framework'
+	@# $(call _build_target,ios/amd64)
+	@# gcc.exe: error: unrecognized command-line option '-framework'
+	@# $(call _build_target,ios/arm64)
+	$(call _build_target,js/wasm)
+	$(call _build_target,linux/386)
+	$(call _build_target,linux/amd64)
+	$(call _build_target,linux/arm)
+	$(call _build_target,linux/arm64)
+	$(call _build_target,linux/mips)
+	$(call _build_target,linux/mips64)
+	$(call _build_target,linux/mips64le)
+	$(call _build_target,linux/mipsle)
+	$(call _build_target,linux/ppc64)
+	$(call _build_target,linux/ppc64le)
+	$(call _build_target,linux/riscv64)
+	$(call _build_target,linux/s390x)
+	$(call _build_target,netbsd/386)
+	$(call _build_target,netbsd/amd64)
+	$(call _build_target,netbsd/arm)
+	$(call _build_target,netbsd/arm64)
+	$(call _build_target,openbsd/386)
+	$(call _build_target,openbsd/amd64)
+	$(call _build_target,openbsd/arm)
+	$(call _build_target,openbsd/arm64)
+	$(call _build_target,openbsd/mips64)
+	$(call _build_target,plan9/386)
+	$(call _build_target,plan9/amd64)
+	$(call _build_target,plan9/arm)
+	$(call _build_target,solaris/amd64)
+	$(call _build_target,windows/386,$(EXEEXT))
+	$(call _build_target,windows/amd64,$(EXEEXT))
+	$(call _build_target,windows/arm,$(EXEEXT))
+	$(call _build_target,windows/arm64,$(EXEEXT))
 
 _compress-all-binaries:
-	for f in $(BINARIES); do      \
-		tar czf $$f.tar.gz $$f;    \
+	for f in bin/*; do      \
+		tar czf - $$f | gzip -9 - > $$f.tar.gz;    \
 	done
-	@rm $(BINARIES)
+	@rm -f bin/*
```

Comment:
With this, you are compressing all binaries and deleting them immediately.
We should keep the compressed binaries.

Reply:
Fixed.